lexer.go

```diff
@@ -15,8 +15,6 @@ var keywords = map[string]int{
 	"NULL":       NULL,
 	"NONE":       NONE,
 	"INTEGER":    INTEGER,
-	"NUMERIC":    NUMERIC,
```

Comment:
Remove keywords from the lexers. That means that the lexer won't emit tokens for those keywords, which will probably result in a syntax error